zulipterminal/helper.py

```diff
@@ -21,7 +21,29 @@
 LINUX = platform.system() == "Linux"
 WSL = 'microsoft' in platform.release().lower()
 
-Message = Dict[str, Any]
+Message = TypedDict('Message', {
```

Comment:
As I mentioned via PM, this would be a good change to make separately. If you know which parameters require the `total=False` then we can also note that, as with python 3.6 we should be able to inherit `TypedDict`s, and remove the need for every parameter to be non-total. This could also be used effectively for API versioning.